### ISSUE TITLE
fill disapears in curved wall

### ISSUE THREAD

**Issue by SavuGust27** [IfcOpenShell/IfcOpenShell#5254]
Since there is no dedicated tool to create curved walls, the way that i have resolve this is by choosing horizontal layers in the type template when creating a type, that way i can edit its shape, and i just add the height of the wall in the material panel, in the layer thickness. it works for me and i like it because i can use the CAD tools to create 3 point curves. so far so good. But i recently notice that when i print it, the fill is not shown. and it is not only for that wall, but for all the others, the fills are not created. 

![Screenshot 2024-08-29 101259](https://github.com/user-attachments/assets/4c25903b-b67a-4e83-899e-fc10013127a9)
part with straight walls

![Screenshot 2024-08-29 101417](https://github.com/user-attachments/assets/b7a01f42-00e3-4f38-a558-485fba0d3ffe)
with the curved walls

So i went ahead and created a curved wall from mesh, which i dint want to do because editing a mesh after you create a curve wall is not the best experience in vanilla blender, and that print with fills

![Screenshot 2024-08-29 101614](https://github.com/user-attachments/assets/57fda365-de78-4c1c-aecc-2c4a68ba15c6)
The curved wall is a mesh created and then assigned into a ifcwall 

any idea why this happens?
i also try using the horizontal layers or the extruded profiles whenever i can instead of creating meshes because in elevation, they print way better

**Comment by Moult** [comment]
Can you share your file?

I just tested and it works fine here. But this is with the new Bisect cut mode. The OCC cut mode is a bit more picky about these things.

[untitled.ifc.txt](https://github.com/user-attachments/files/16834426/untitled.ifc.txt)

![2024-09-02-180247_199x228_scrot](https://github.com/user-attachments/assets/9727d9d9-3cb9-4c76-9f7a-f7cfb865b52d)

This is with bisect:

![2024-09-02-180128_102x64_scrot](https://github.com/user-attachments/assets/a088913e-bae5-43cc-b111-01ab9193f1b0)

This is with OCC:

![2024-09-02-180152_91x56_scrot](https://github.com/user-attachments/assets/0c0958ed-0b89-4c9e-bae6-34a9a28efaa8)

**Comment by SavuGust27** [comment]
yes, tried it again with the new realease, and with bisect it works great, openCASCADE keep printing with no fill on my end

**Comment by theoryshaw** [comment]
@SavuGust27 can you provide an example file to test?
